README.md

```diff
@@ -2,7 +2,11 @@
 
 [![Gitter](https://badges.gitter.im/hail-is/hail.svg)](https://gitter.im/hail-is/hail?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![CI Status](https://ci.hail.is/app/rest/builds/buildType:(id:HailSourceCode_HailCi)/statusIcon)](https://ci.hail.is/viewType.html?buildTypeId=HailSourceCode_HailCi&branch_HailSourceCode_HailMainline=%3Cdefault%3E&tab=buildTypeStatusDiv)
 
-[Hail](https://hail.is) is an open-source, scalable framework for exploring and analyzing genomic data. Starting from genetic data in [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf), [BGEN](http://www.well.ox.ac.uk/~gav/bgen_format/bgen_format_v1.2.html) or [PLINK](https://www.cog-genomics.org/plink2/formats) format, Hail can, for example:
+[Hail](https://hail.is) is an open-source, scalable framework for exploring and analyzing genomic data. 
+
+The Hail project began in Fall 2015 to empower the worldwide genetics community to [harness the flood of genomes](https://www.broadinstitute.org/blog/harnessing-flood-scaling-data-science-big-genomics-era) to discover the biology of human disease. Hail has been used for dozens of major studies and is the core analysis platform of large-scale genomics efforts such as [gnomAD](http://gnomad.broadinstitute.org/).
+
+Starting from genetic data in [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf), [BGEN](http://www.well.ox.ac.uk/~gav/bgen_format/bgen_format_v1.2.html) or [PLINK](https://www.cog-genomics.org/plink2/formats) format, Hail can, for example:
```

Comment:
I think these paragraphs could be revamped for 0.2, but I guess let's wait until its not beta.

Reply:
I'd be okay updating for 0.2 now.

Reply:
I'm not sure what functionality from 0.2 we'd like to highlight here. If someone else wants to suggest changes to these bullet points, feel free. Otherwise, I just made a couple changes.